_topics/en/erlay.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Erlay
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - P2P Network Protocol
+  - Bandwidth Reduction
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+excerpt: >
+  **Erlay** is proposal to use set reconciliation between Bitcoin full
```

Comment:
I think it would be more accurate to say "...is a proposal to relay unconfirmed transactions in two phases: a low-fanout flooding phase and a set reconciliation phase."

Reply:
Done